README.md

```diff
@@ -1 +1,172 @@
-# figma_layout_grid
+# Figma layout grid
+```figma_layout_grid``` is a Flutter package which imitates Figma's layout grid and all its functionalities. It helps you determine whether your UI components match the design file and align properly. 
```

Comment:
In whole README.md file there are code spans which could be wrote with a single back-tick sign (`` ` ``) instead of triple signs (`` ``` ``) e.g. `` `figma_layout_grid` `` instead of `` ```figma_layout_grid``` ``. 
The visual result will be the same, but sticking to the most popular convention where single back-ticks are used for inline code (within the paragraph) and triple back-ticks to the code blocks, will be the most convenient for the rest of potential developers who will put their hands on editing this README.md in the future